README.md

```diff
@@ -1,6 +1,47 @@
-Redmine Zulip plugin
-====================
+# Redmine Zulip Plugin
 
-Get notifications about new and updated issues directly in Zulip!
+Get Zulip notifications for your Redmine issues!
 
-Please visit https://zulipchat.com/integrations/#redmine for detailed installation instructions.
+## Installing
+
+0. Please navigate to your Redmine instance's root directory by running:
+
+```sh
+cd /path/to/redmine
+```
+
+1. Clone the **Redmine Zulip Plugin** into `plugins` directory
+
+```sh
+git clone https://github.com/zulip/zulip-redmine-plugin.git plugins/redmine_zulip
+```
+
+2. **Restart** your Redmine instance
+
+3. Update the Redmine database by running:
+
+```sh
+rake redmine:plugins:migrate
+```
+
+## Configuring plugin settings
+
+#### Global settings
+
+Log into your Redmine instance, click on **Administration** in the top-left corner, then click on **Plugins**.
+
+Find the **Redmine Zulip** plugin, and click **Configure**. You must now set the following:
+
+* Zulip API URL (e.g `https://yourZulipDomain.zulipchat.com/api`)
```

Comment:
I think interface-wise, it'd be cleaner to just ask for them to specify the Zulip server URL (e.g. `https://yourZulipDomain.zulipchat.com`) and have the plugin code add the /api part.  

(The original plugin had this interface because the Zulip API model was originally hardcoded to e.g. `api.zulip.com/v1` in like 2013, but Zulip has standardized on the /api URL system in like 2014)